nomad/grapl-core.nomad

```diff
@@ -93,24 +117,30 @@ variable "node_identifier_tag" {
 }
 
 variable "node_identifier_queue" {
-  type    = string
-  default = "http://localhost:9324/000000000000/node-identifier-queue"
+  type = string
 }
 
 variable "node_identifier_dead_letter_queue" {
-  type    = string
-  default = "http://localhost:9324/000000000000/node-identifier-dead-letter-queue"
+  type = string
```

Comment:
It would be great if all the variables had descriptions, particularly with these queues, since it's easy to mix them up, depending on which service they are being passed to.